.gitignore

```diff
@@ -24,3 +24,7 @@ coverage
 .talismanrc
 
 .idea
+
+# credentials
+config/credentials.yml.enc
+/config/master.key
```

Comment:
master.key path is absolute. It needs the first slash removed

Reply:
I have already merged this in ready for today's deployment so will do this as part of the next pull request.